config/src/lib.rs

```diff
@@ -451,6 +451,45 @@ impl Committee {
             .collect()
     }
 
+    /// Return all the network addresses in the committee.
+    fn get_all_network_addresses(&self) -> HashSet<&Multiaddr> {
+        self.authorities
+            .values()
+            .map(|authority| {
+                std::iter::once(&authority.primary.primary_to_primary)
+                    .chain(std::iter::once(&authority.primary.worker_to_primary))
+                    .chain(
+                        authority
+                            .workers
+                            .values()
+                            .map(|address| &address.transactions),
+                    )
+                    .chain(
+                        authority
+                            .workers
+                            .values()
+                            .map(|address| &address.worker_to_worker),
+                    )
+                    .chain(
+                        authority
+                            .workers
+                            .values()
+                            .map(|address| &address.primary_to_worker),
+                    )
+            })
+            .flatten()
+            .collect()
+    }
+
+    /// Return the network addresses that are present in the current committee but that are absent
+    /// from the new committee (provided as argument).
+    pub fn network_diff<'a>(&'a self, new_committee: &'a Self) -> HashSet<&Multiaddr> {
```

Comment:
Nit: `new` and `old` are concepts for the caller of this method. The typical way to present things in such a method is `self` and `other` (for the argument).